Package.swift

```diff
@@ -195,6 +195,18 @@ let package = Package(
       name: "TokamakTestRenderer",
       dependencies: ["TokamakCore"]
     ),
+    .testTarget(
+      name: "TokamakLayoutTests",
```

Comment:
Are these new tests running on CI? I don't see any changes to GHA workflows in this diff.

Reply:
I think the workflow runs every test target in the package already.

Reply:
Ah, that's great! Sorry about the confusion.